e2e/e2etests/e2etests.go

```diff
@@ -1330,6 +1331,12 @@ var AllE2ETests = []runner.E2ETest{
 		[]runner.ArgDefinition{},
 		TestInboundTrackers,
 	),
+	runner.NewE2ETest(
+		TestOpcodesName,
+		"test opcodes support in ZEVM",
```

Comment:
consider adding which opcodes to the description

Reply:
I prefer keeping a single test to test any opcode we want to test in the future, we will just need to run it once